docs/src/components/AppDrawer.js

```diff
@@ -103,17 +113,21 @@ export default class AppDrawer extends Component {
         docked={this.props.docked}
       >
         <div className={classes.nav}>
-          <Toolbar>
+          <Toolbar className={classes.toolbar}>
             <Link className={classes.title} to="/" onClick={this.props.onRequestClose}>
               <Text type="title" gutterBottom>
                 Material UI
               </Text>
-              {process.env.MATERIAL_UI_VERSION ? (
-                <Text type="caption">
-                  {`(v${process.env.MATERIAL_UI_VERSION})`}
-                </Text>
-              ) : null}
             </Link>
+            {process.env.MATERIAL_UI_VERSION ? (
+              <Text type="caption">
+                (<a
```

Comment:
Bah, forgot to re-run the tests!

> Why using a `(` here?

Because you used one? 😄 I moved it outside the anchor so that it wasn't part of the linked text. Now I can't decide if it should be part of the link or not!  

Having done that, I couldn't decide on the best way to format the code (linting aside).